src/test/java/racingcar/CarTest.java

```diff
@@ -2,20 +2,38 @@
 
 import org.junit.jupiter.api.Test;
 
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
 import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.AssertionsForClassTypes.assertThatThrownBy;
 
 class CarTest {
 
     @Test
-    void 자동차는_경기_play_횟수를_알_수_있다() {
-        Car car = new Car();
+    void 자동차는_이름을_기준으로_중복된_객체를_제거한다() {
+        Set<Car> cars = new HashSet<>(List.of(new Car("a"), new Car("a"), new Car("b")));
+        assertThat(cars).hasSize(2);
```

Comment:
```suggestion
        Racing racing = new Racing(List.of(new Car("a"), new Car("a"), new Car("b")));

        assertThat(racing.contain(new Car("a"))).isTrue();
```

객체에게 물어본다면 어떨까요? 해당 방법은 예시일뿐 객체에게 메시지를 보내고
그리고, 메소드를 테스트 하는것을 경험해보면 좋겠습니다.